Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeUpdates.ps1

```diff
@@ -17,7 +21,7 @@ function Get-ExchangeUpdates {
         $RegLocation = "SOFTWARE\Microsoft\Updates\Exchange 2019"
     }
 
-    $RegKey = Get-RemoteRegistrySubKey -MachineName $Script:Server `
+    $RegKey = Get-RemoteRegistrySubKey -MachineName $Server `
```

Comment:
Not a hard requirement but as we do these many changes, we could do splatting here too.